pino.js

```diff
@@ -272,6 +272,17 @@ Object.defineProperty(pinoPrototype, 'isLevelEnabled', {
   value: isLevelEnabled
 })
 
+function getPrettyStream (opts) {
+  var prettyFactory
+  var prettyModule = opts.prettifier || 'pino-pretty'
+  try {
+    prettyFactory = require(prettyModule)
+    return prettyFactory(opts).asMetaWrapper(process.stdout)
```

Comment:
I think we might as well include asMetaWrapper in here instead. I do not see a reason why a prettifier might not want that.

Reply:
I don't understand this statement. Can you reword to clarify?